hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/spi/impl/ReconDBDefinition.java

```diff
@@ -74,6 +74,16 @@ public ReconDBDefinition(String dbName) {
           NSSummary.class,
           new NSSummaryCodec());
 
+  // Container Replica History with bcsId tracking.
+  public static final DBColumnFamilyDefinition
+      <Long, ContainerReplicaHistoryList> REPLICA_HISTORY_V2 =
+      new DBColumnFamilyDefinition<Long, ContainerReplicaHistoryList>(
+          "replica_history_v2",
```

Comment:
So in a future Recon upgrade process we should probably drop (or the upgrade layout of) `REPLICA_HISTORY` (V1) table right? to avoid wasted storage space.

Reply:
Yes, we can clean up in the future.